README.md

```diff
@@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
```

Comment:
Regardless of the past commits, this would obviously be for future commits.
On Sep 9, 2015 19:38, "Vincent Batts" <vbatts@hashbangbash.com> wrote:

> Jesus Trevor... I ....
> On Sep 9, 2015 19:37, "W. Trevor King" <notifications@github.com> wrote:
>
>> In README.md
>> <https://github.com/opencontainers/specs/pull/168#discussion_r39110526>:
>>
>> > @@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
>> >
>> >  You can add the sign off when creating the git commit via `git commit -s`.
>> >
>> > +### Commit Style
>> > +
>> > +Simple house-keeping for clean git history.
>> > +
>> > +1. Separate subject from body with a blank line
>> > +2. Limit the subject line to 50 characters
>>
>> And because it's been too long since I've busted out some Matplotlib,
>> here's a histogram for git.git:
>>
>> [image: x]
>> <https://cloud.githubusercontent.com/assets/209920/9776781/d9e8bb6a-5710-11e5-9fd1-a7b62e54b677.png>
>>
>> Generated with:
>>
>> import sysimport matplotlib.pyplot
>> lines = sys.stdin.read().splitlines()
>> lengths = [len(line) for line in lines]
>> figure = matplotlib.pyplot.figure()
>> axes = figure.add_subplot(1, 1, 1)
>> axes.hist(lengths, 50)
>> axes.set_xlabel('commit message length (chars)')
>> axes.set_ylabel('counts in git.git through v2.3.4')
>> figure.savefig('x.png')
>>
>> ;).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/opencontainers/specs/pull/168/files#r39110526>.
>>
>